js/search-algorithm.js

```diff
@@ -114,22 +114,439 @@ define([
+            '\u2DA8-\u2DAE',
+            '\u2DB0-\u2DB6',
+            '\u2DB8-\u2DBE',
+            '\u2DC0-\u2DC6',
+            '\u2DC8-\u2DCE',
+            '\u2DD0-\u2DD6',
+            '\u2DD8-\u2DDE',
+            '\u2E2F',
+            '\u3005',
+            '\u3006',
+            '\u3031-\u3035',
+            '\u303B',
+            '\u303C',
+            '\u3041-\u3096',
+            '\u309D-\u309F',
+            '\u30A1-\u30FA',
+            '\u30FC-\u30FF',
+            '\u3105-\u312D',
+            '\u3131-\u318E',
+            '\u31A0-\u31BA',
+            '\u31F0-\u31FF',
+            '\u3400-\u4DB5',
+            '\u4E00-\u9FCC',
+            '\uA000-\uA48C',
+            '\uA4D0-\uA4FD',
+            '\uA500-\uA60C',
+            '\uA610-\uA61F',
+            '\uA62A',
+            '\uA62B',
+            '\uA640-\uA66E',
+            '\uA67F-\uA697',
+            '\uA6A0-\uA6E5',
+            '\uA717-\uA71F',
+            '\uA722-\uA788',
+            '\uA78B-\uA78E',
+            '\uA790-\uA793',
+            '\uA7A0-\uA7AA',
+            '\uA7F8-\uA801',
+            '\uA803-\uA805',
+            '\uA807-\uA80A',
+            '\uA80C-\uA822',
+            '\uA840-\uA873',
+            '\uA882-\uA8B3',
+            '\uA8F2-\uA8F7',
+            '\uA8FB',
+            '\uA90A-\uA925',
+            '\uA930-\uA946',
+            '\uA960-\uA97C',
+            '\uA984-\uA9B2',
+            '\uA9CF',
+            '\uAA00-\uAA28',
+            '\uAA40-\uAA42',
+            '\uAA44-\uAA4B',
+            '\uAA60-\uAA76',
+            '\uAA7A',
+            '\uAA80-\uAAAF',
+            '\uAAB1',
+            '\uAAB5',
+            '\uAAB6',
+            '\uAAB9-\uAABD',
+            '\uAAC0',
+            '\uAAC2',
+            '\uAADB-\uAADD',
+            '\uAAE0-\uAAEA',
+            '\uAAF2-\uAAF4',
+            '\uAB01-\uAB06',
+            '\uAB09-\uAB0E',
+            '\uAB11-\uAB16',
+            '\uAB20-\uAB26',
+            '\uAB28-\uAB2E',
+            '\uABC0-\uABE2',
+            '\uAC00-\uD7A3',
+            '\uD7B0-\uD7C6',
+            '\uD7CB-\uD7FB',
+            '\uF900-\uFA6D',
+            '\uFA70-\uFAD9',
+            '\uFB00-\uFB06',
+            '\uFB13-\uFB17',
+            '\uFB1D',
+            '\uFB1F-\uFB28',
+            '\uFB2A-\uFB36',
+            '\uFB38-\uFB3C',
+            '\uFB3E',
+            '\uFB40',
+            '\uFB41',
+            '\uFB43',
+            '\uFB44',
+            '\uFB46-\uFBB1',
+            '\uFBD3-\uFD3D',
+            '\uFD50-\uFD8F',
+            '\uFD92-\uFDC7',
+            '\uFDF0-\uFDFB',
+            '\uFE70-\uFE74',
+            '\uFE76-\uFEFC',
+            '\uFF21-\uFF3A',
+            '\uFF41-\uFF5A',
+            '\uFF66-\uFFBE',
+            '\uFFC2-\uFFC7',
+            '\uFFCA-\uFFCF',
+            '\uFFD2-\uFFD7',
+            '\uFFDA-\uFFDC'
+        ];
+
+        wordCharacters = wordCharacters.join('');
+		
+		var search = _.extend({
```

Comment:
could you sort out the indentation here? it wasn't right to start with but we may as well correct it whilst you're here.